tests/system/src/test/java/test/javafx/stage/StageMixedSizeTest.java

```diff
@@ -96,7 +98,9 @@ private void createStage(BiConsumer<Stage, StackPane> beforeShown,
             }
 
             mainStage.setScene(new Scene(sp));
+
             mainStage.setOnShown(e -> {
+                Util.sleep(500);
```

Comment:
We don't want to sleep on the FX application thread. I think you'll need to split this into separate `Runnable`s for separate calls to `Util::runAndWait` and have the sleeps between them.

Reply:
I did it with a Timeline, is it ok?

Reply:
That should be OK. I'll test it tomorrow.